dist/image-sequencer.js

```diff
@@ -47721,6 +47721,25 @@ ImageSequencer = function ImageSequencer(options) {
     else modulesdata = modules[name][1];
     return modulesdata;
   }
+
+  function toString(step_name,options){
+    if(step_name){
+      return `${step_name}(${Object.keys(options).map(key=>key + ':' + options[key]).join(',')})`
```

Comment:
Interesting -- i'm not familiar with this style of quotein JS -- is it an ES6 thing?  I guess I'd prefer to do this over a few steps so it's easier to read and maintain, is that OK?

Reply:
Yeah @jywarren Actually doing it this way we save the `+` and gives us the ability to use `'` and `"` without escaping

Reply:
how this works is that inside `\`\`` if we want to evaluate an expression inline we have to use ${} and this gets replaced by evaluated value

Reply:
I just think we could put `Object.keys(options).map(key=>key + ':' + options[key]).join(',')` on its own line like `var configurations = ...`

Reply:
@jywarren I'm sorry I didn't quite get this, so I should just break the 3 parts, right?